improver/threshold.py

```diff
@@ -295,7 +295,7 @@ def _set_thresholds(
             # unit conversion.
             if isinstance(threshold_values, numbers.Number):
                 threshold_values = [threshold_values]
-            thresholds = [float(x) for x in threshold_values]
+            thresholds = [np.float32(x) for x in threshold_values]
```

Comment:
You've not noticed the specific comment here that says the float types are made 64 bit to avoid rounding errors during unit conversion.

If you leave these as float64, does that change your acceptance test data further?

Reply:
Do we know that the change is "wrong"? If it is not wrong, then I suggest we accept the change in the acceptance test data and leave this as float64.